lib/message_bus/backends/active_record.rb

```diff
@@ -0,0 +1,205 @@
+# frozen_string_literal: true
+
+require 'message_bus/backends/postgres'
+require 'message_bus/rails/models/message'
+require 'message_bus/codec/itself'
+require 'message_bus/pub_sub/redis'
+
+module MessageBus
+  module Backends
+    class ActiveRecord < Postgres
+      class Client
+        class Listener
+          attr_reader :do_sub, :do_unsub, :do_message
+
+          def subscribe(&block)
+            @do_sub = block
+          end
+
+          def unsubscribe(&block)
+            @do_unsub = block
+          end
+
+          def message(&block)
+            @do_message = block
+          end
+        end
+
+        class << self
+          def reset!(config)
+            self::Client.new(config).reset!
+          end
+        end
+
+        def initialize(config)
+          @config = config
+          @listening_on = {}
+          @available = []
+          @subscribe_connection = nil
+          @subscribed = false
+          @mutex = Mutex.new
+        end
+
+        # @param channel [String]
+        # @param value [Hash]
+        # @return [Integer] persisted record id
+        def add(channel, value)
+          Rails::Message.create(channel: channel, value: value).id
```

Comment:
`#id` will be just `nil`. Besides, we don't have any rails validations, so `create` can't fail silently - in case of db error there will be an exception